prov/efa/test/efa_unit_test_fork_support.c

```diff
@@ -0,0 +1,33 @@
+#include "efa_unit_tests.h"
+
+/**
+ * @brief verify efa_fork_support_request_initialize() set value of g_efa_fork_status and efa_env.use_huge_page correctly
+ */
+void test_efa_fork_support_request_initialize_when_ibv_fork_support_is_needed(void **state)
+{
+	setenv("FI_EFA_FORK_SAFE", "1", true);
+	g_efa_unit_test_mocks.ibv_is_fork_initialized = &efa_mock_ibv_is_fork_initialize_return_disabled;
```

Comment:
Curious why we didn't use a single mock function and utilize `will_return` to change the return value?

Reply:
Should have used `will_return` and `mock`. Latest version did that